src/main/kotlin/com/mineinabyss/staminaclimb/Extensions.kt

```diff
@@ -43,6 +46,14 @@ fun Player.launchInDirection() {
 
 const val checkRange = 0.4 //TODO config
 
+val Player.equipmentModifiers: Float
```

Comment:
You could alternatively write a listener that updates a component on the player when any items changes but I think this would be mroe complicated. You could do this quite nicely with a query but I think it would perform worse than what you're doing right now so this is fine.

Just move out the `player?.inventory?.toGeary()` since you're recalcualting it for every item.

Also I think the modifier should be multiplicative, not additive. If we want the option to do both, we have to go down the route attribute modifiers do in vanilla.